### ISSUE TITLE
Scrolling upwards laggards behind

### ISSUE THREAD

**Issue by KeenanLukeOM** [DXgovernance/infinite-hackathon#37]
When scrolling up on the Genesis page, the background can't seem to keep up with the user's scroll. This causes black bars to appear when scrolling up. If a solution is easy to implement, it would be great to have it for a final build. 

![chrome_h8UOrufhoF](https://user-images.githubusercontent.com/76129769/191636648-da25c4f7-2c1a-47b7-bb81-88c3e8afed60.gif)

**Comment by KeenanLukeOM** [comment]
Still experiencing this issue!

**Comment by rulfo71** [comment]
can you show me where? i'm not seing happen since there is a black section above the ring. 
are you trying on this link? 
https://rulfo71.github.io/infinite-hackathon/#/genesis

**Comment by KeenanLukeOM** [comment]
Correct! I am on [this link](https://rulfo71.github.io/infinite-hackathon/#/genesis), it's definitely dimmer now, but still there. This also appeared to be the change that resulted in the orange ring being moved down in focus?

![chrome_vGgUW34c6N](https://user-images.githubusercontent.com/76129769/192428008-29da1e35-cc0c-4369-82d7-130a2beafbc9.gif)
